### ISSUE TITLE
Catch errors on WalletConnect.enable

### ISSUE THREAD

**Issue by pakokrew** [Web3Modal/web3modal#156]
As there's no try/catch around `await provider.enable()`, if the user closes the WalletConnect modal, there's an uncaught exception, and that makes web3modal stuck on this cached provider.

https://github.com/Web3Modal/web3modal/blob/4a617060b93b5602f2e6a81a281f3e95194c29bf/src/providers/connectors/walletconnect.ts#L39

```
Uncaught (in promise) Error: User closed WalletConnect modal
    at index.js:144
    at HTMLDivElement.<anonymous> (index.js:42)
```

**Comment by pedrouid** [comment]
Ouch... good catch! Going to publish a fix asap

**Comment by pedrouid** [comment]
Fix published on v1.6.2

**Comment by pakokrew** [comment]
That was fast 👌
